commit 4375e9c00dd0ffb745d85cd2fc0b26afbea0c0bf
Author: Marcus Meissner <marcus@jet.franken.de>
Date:   Sat Oct 20 19:24:23 2018 +0200

    disable manifest read. apple does not do what is in ptp 1.1

diff --git a/camlibs/ptp2/ptp.c b/camlibs/ptp2/ptp.c
index 5327865..fb32483 100644
--- a/camlibs/ptp2/ptp.c
+++ b/camlibs/ptp2/ptp.c
@@ -2570,6 +2570,7 @@ ptp_list_folder (PTPParams *params, uint32_t storage, uint32_t handle) {
 		/*debug_objectinfo(params, handle, &ob->oi);*/
 	}
 
+#if 0 /* apple devices report it, but the conrtent they have does not match the standard somehow. Neesd further debugging */
 	if (ptp_operation_issupported(params, PTP_OC_GetFilesystemManifest)) {
 		uint64_t		numoifs = 0;
 		PTPObjectFilesystemInfo	*oifs = NULL;
@@ -2636,6 +2637,8 @@ ptp_list_folder (PTPParams *params, uint32_t storage, uint32_t handle) {
 		if (changed) ptp_objects_sort (params);
 		return PTP_RC_OK;
 	}
+#endif
+
 fallback:
 	ptp_debug (params, "Listing ... ");
 	if (handle == 0) xhandle = PTP_HANDLER_SPECIAL; /* 0 would mean all */
